Dockerfile

```diff
@@ -44,7 +45,11 @@ ENV RUSTUP_HOME=/opt/rustup \
     PATH=/cargo/bin:$PATH
 COPY dev-tools.json dev-tools.json
 COPY scripts/setup scripts/setup-cargo-binstall scripts/setup-rust scripts/
-RUN scripts/setup rust cargo-binstall candid-extractor ic-wasm
+RUN scripts/setup rust
```

Comment:
We  should not pin version here? Particularly didc and ic-wasm?

Reply:
The versions are pinned - in dev-tools.json

Reply:
setup looks in that file for version information.